3scale/porta
# [WIP] Appdux 765 plan selector

**What this PR does / why we need it**:

This replaces the old plan select control on the application details page with the newer React plan select control.
It is WIP because the current setting of the plan is not being shown properly.

**Which issue(s) this PR fixes** 

APPDUX-765

**Verification steps** 

For an application, check that the list of plans is presented and correct. Check that the selected plan is saved correctly (in progress)
For application plans, check that the default plan select control still works and correctly sets the default plan
For any dashboard cards, check that there is no overflow other than an open dropdown menu.

**Special notes for your reviewer**:
